specification/relay/resource-manager/Microsoft.Relay/stable/2021-11-01/examples/NameSpaces/RelayNameSpaceDelete.json

```diff
@@ -0,0 +1,13 @@
+{
+  "parameters": {
+    "namespaceName": "example-RelayNamespace-5849",
+    "resourceGroupName": "SouthCentralUS",
+    "api-version": "2018-01-01-preview",
```

Comment:
>"2018-01-01-preview" [](http://example.com/codeflow?start=19&length=20)

update api-version, make sure to update all your examples.

Reply:
done